fmriprep/data/itkIdentityTransform.tfm

```diff
@@ -0,0 +1,5 @@
+#Insight Transform File V1.0
+#Transform 0
+Transform: MatrixOffsetTransformBase_double_3_3
+Parameters: 1 0 0 0 1 0 0 0 1 0 0 0
+FixedParameters: 0 0 0
```

Comment:
I think this file will need to be added to `setup.py`?

Reply:
yes